electrumsv/tests/test_wallet.py

```diff
@@ -666,8 +906,13 @@ def check_specific_wallets(wallet: Wallet, password: str, storage_info: WalletSt
         account_descriptions.append([account_description_row.account_id,
             hash_to_hex_str(account_description_row.tx_hash), account_description_row.description])
 
-   # Account 1 is the pre-existing account (and predates the petty cash account).
-    account = wallet.get_account(1)
+    # Get the non-petty-cash account
```

Comment:
I don't understand what is going on here. This comment is useless. You need to write WHY you are doing something. Just writing a literal description of what a statement does adds nothing. And now you're excluding a case from test coverage for no discernable reason.